drivers/mysql.go

```diff
@@ -313,6 +320,7 @@ func (db *MySQL) ExecuteQuery(query string) (results [][]string, err error) {
 
 // TODO: Rewrites this logic to use the primary key instead of the id
 func (db *MySQL) UpdateRecord(table, column, value, primaryKeyColumnName, primaryKeyValue string) error {
+	table = db.formatTableName(table)
 	query := fmt.Sprintf("UPDATE %s SET %s = \"%s\" WHERE %s = \"%s\"", table, column, value, primaryKeyColumnName, primaryKeyValue)
```

Comment:
Also, I'm very worried about the  possible SQL injection

You should consider using `?` for value, then pass `value ` to Exec as an arg